### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by maltesander**

### CODE REVIEW

File: crd/src/lib.rs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+use stackable_operator::Crd;
+
+#[derive(
+    Clone, CustomResource, Debug, Default, Deserialize, Eq, Hash, JsonSchema, PartialEq, Serialize,
+)]
+#[kube(
+    group = "opa.stackable.tech",
+    version = "v1",
+    kind = "OpaServer",
+    shortname = "os",
```
Comment (soenkeliebau) [inline crd/src/lib.rs:13]: nit: wouldn't "opa" be the obvious choice here?

File: server/packaging/rpm/SOURCES/stackable-opa-operator-server-VERSION/usr/lib/systemd/system/stackable-opa-operator-server.service [hunk 2]
```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Stackable OpenPolicyAgent Operator
+Before=
+After=network.target
+[Service]
+User=root
+ExecStart=/opt/stackable/opa-operator/opa-operator-server
```
Comment (soenkeliebau) [inline server/packaging/rpm/SOURCES/stackable-opa-operator-server-VERSION/usr/lib/systemd/system/stackable-opa-operator-server.service:7]: The binary here needs to match the parameter you pass to the buildrpm script in the nightly action.

File: server/packaging/debian/service [hunk 3]
```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Stackable OpenPolicyAgent Operator
+Before=
+After=network.target
+[Service]
+User=root
+ExecStart=/opt/stackable/opa-operator/opa-operator-server
```
Comment (soenkeliebau) [inline server/packaging/debian/service:7]: See comment in other service file about the binary name.

**Review by soenkeliebau — CHANGES_REQUESTED** [review]
Left two comments (actually the same comment, but in two places) :)

File: crd/src/lib.rs (hunk 1 above)
Comment (maltesander) [reply crd/src/lib.rs:13]: Yeah, any changes in the group name then? 
"opa.opa.stackable.tech" is not my favorite :)

File: server/packaging/rpm/SOURCES/stackable-opa-operator-server-VERSION/usr/lib/systemd/system/stackable-opa-operator-server.service (hunk 2 above)
Comment (maltesander) [reply server/packaging/rpm/SOURCES/stackable-opa-operator-server-VERSION/usr/lib/systemd/system/stackable-opa-operator-server.service:7]: Thanks will change.

**Review by soenkeliebau — APPROVED** [review]
LGTM

**Pull request merged**